crates/cdk-mintd/src/loggers.rs

```diff
@@ -0,0 +1,117 @@
+use chrono::Utc;
+use reqwest::Client;
+use serde::Serialize;
+use std::fmt::Write;
+use tokio::sync::mpsc::{self, Sender};
+use tokio::task::JoinHandle;
+use tracing::field::{Field, Visit};
+use tracing::Event;
+use tracing_subscriber::layer::{Context, Layer};
+use tracing_subscriber::registry::LookupSpan;
```

Comment:
```suggestion
use std::fmt::Write;

use chrono::Utc;
use reqwest::Client;
use serde::Serialize;
use tokio::sync::mpsc::{self, Sender};
use tokio::task::JoinHandle;
use tracing::field::{Field, Visit};
use tracing::Event;
use tracing_subscriber::layer::{Context, Layer};
use tracing_subscriber::registry::LookupSpan;
```

Std imports should be first, this will fix formatting ci